change-developers-mindset.md

```diff
@@ -0,0 +1,49 @@
+## Change the developers mindset
+
+## Context
+
+* Top-down InnerSource support exists.
+* 3k population of developers.
+* Middle Management is not supporting InnerSource.
+* There's already a successful InnerSource group in the early stages.
+* Code visibility is product dependent.
+
+## Problem  
+
+* How do you change developer mindset? It's difficult to push developers to do things.
+* Developers are resisting the change, they are in their comfort zone and it's hard to get them to go out of it.
+* Developer organization's maturity is high, so people are used to being in some hierarchy/rules.
+* Developers are formed in agile, the shift from agile is difficult
+
+## Forces  
+
+* Managers are previous developers, so they like the way they were promoted and they want to proceed in similar ways.
+* Managers restrict what developers can spend time on.
+* Top-down approach to the InnerSource.
```

Comment:
This may be more of a context? Unless the force addresses why InnerSource is being approached top down rather from both the top and from a grassroots level.

Reply:
Resolving as the pattern in this PR has changed in the meantime, i.e. it isn't clear if this feedback is still valid. Please review the latest version again.